api/v1alpha1/api.go

```diff
@@ -61,7 +61,7 @@ type AIGatewayRouteSpec struct {
 	// modifications to achieve the necessary jobs, notably inserting the AI Gateway filter responsible for
 	// the transformation of the request and response, etc.
 	//
-	// In the matching conditions in the AIGatewayRouteRule, `x-envoy-ai-gateway-model` header is available
+	// In the matching conditions in the AIGatewayRouteRule, `x-eaig-model` header is available
```

Comment:
tbh, `eaig` looks like a typo unfortunately, suggest `x-ai-eg-model` which reads better?

Reply:
fixed

Reply:
let's stick to ai-eg-* from now!